src/utils/runtime/src/lib.rs

```diff
@@ -63,7 +63,8 @@ fn configure_risingwave_targets_fmt(targets: filter::Targets) -> filter::Targets
         .with_target("risingwave_compute", Level::INFO)
         .with_target("risingwave_compactor", Level::INFO)
         .with_target("risingwave_hummock_sdk", Level::INFO)
-        .with_target("pgwire", Level::ERROR)
+        // set the logging level to DEBUG for pgwire to track all queries when testing.
+        .with_target(PGWIRE_QUERY_LOG, Level::DEBUG)
```

Comment:
We use `release_max_level_debug` so it will be included in production.

You may use `#[cfg(debug_assertions)]` to turn it on instead.

Reply:
Oops, I might be wrong. `release_max_level_debug` just eliminates all log level > DEBUG, but we can still tune log level at runtime by e.g., `RUST_LOG`. So your solution should be correct. 🥵

BTW, what's our settings for release log level? Do we set `RUST_LOG`?

Reply:
It seems by default, i.e., if `RUST_LOG` is not set, all logs defined by `filter` are shown (including DEBUG level). And `RUST_LOG` can be used to supress the logs.

I guess in production we didn't set `RUST_LOG`, so the query log will be included by default.

Reply:
https://github.com/risingwavelabs/risingwave/pull/7016 `RUST_LOG` was added to make CI logging less verbose. Not sure whether it's used by production.

Reply:
How about adding some extra `log` related documentation into  https://github.com/risingwavelabs/risingwave/blob/main/docs/developer-guide.md#logging in this PR?

_Originally posted by @lmatz in https://github.com/risingwavelabs/risingwave/issues/7070#issuecomment-1366444524_
            
🤡